RecoPixelVertexing/PixelTriplets/interface/GPUCACell.h

```diff
@@ -4,8 +4,8 @@
 #ifndef GPU_CACELL_H_
 #define GPU_CACELL_H_
 
-#include "GPUHitsAndDoublets.h"
-#include "RecoLocalTracker/SiPixelRecHits/plugins/siPixelRecHitsHeterogeneousProduct.h"
+#include "RecoPixelVertexing/PixelTriplets/interface/GPUHitsAndDoublets.h"
```

Comment:
To me it looks like this include is not needed. Actually nothing from `GPUHitsAndDoublets.h` appears to be used at the moment?